src/databaseService/index.js

```diff
@@ -50,7 +50,6 @@ const add = async (collectionPath, data) => {
 
 export const usernameAlreadyInUse = async (username, collectionPath = 'users') => {
   const email = await queryEmailFromUsername(username, collectionPath)
-
```

Comment:
Please rollback this. We always introduce blank lines before the return clause unless it's the only instruction.